commands/completion.go

```diff
@@ -0,0 +1,94 @@
+/*
+Copyright © 2022 Symbiosis
+*/
+package commands
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/spf13/cobra"
+	"github.com/symbiosis-cloud/cli/pkg/symcommand"
+	"github.com/symbiosis-cloud/symbiosis-go"
+)
+
+type CompletionCommand struct {
+	Client      *symbiosis.Client
+	CommandOpts *symcommand.CommandOpts
+}
+
+var CompletionCommands = []string{"node-types", "regions", "roles"}
```

Comment:
This looks incorrect

Reply:
> This looks incorrect

Indeed, that was left over from another command that I used as a template.